provider/oauth2/models.py

```diff
@@ -40,14 +40,14 @@ class Meta:
         # See https://code.djangoproject.com/ticket/23348
         app_label = "oauth2"
 
-    user = models.ForeignKey(AUTH_USER_MODEL, related_name='oauth2_client',
-                             blank=True, null=True)
+    user = models.ForeignKey(AUTH_USER_MODEL, related_name='oauth2_client', blank=True, null=True)
     name = models.CharField(max_length=255, blank=True)
     url = models.URLField(help_text="Your application's URL.")
     redirect_uri = models.URLField(help_text="Your application's callback URL")
     client_id = models.CharField(max_length=255, default=short_token)
     client_secret = models.CharField(max_length=255, default=long_token)
     client_type = models.IntegerField(choices=CLIENT_TYPES)
+    logout_uri = models.URLField(help_text="Your application's logout URL", null=True, blank=True)
```

Comment:
URLField is a CharField for a URL. From the Django [docs](https://docs.djangoproject.com/en/1.9/ref/models/fields/#null):
> Avoid using null on string-based fields such as CharField and TextField because empty string values will always be stored as empty strings, not as NULL. If a string-based field has null=True, that means it has two possible values for “no data”: NULL, and the empty string. In most cases, it’s redundant to have two possible values for “no data;” the Django convention is to use the empty string, not NULL.